kahuna/public/js/upload/view.html

```diff
@@ -27,5 +27,6 @@ <h2 class="flex-spacer">Your past 50 uploads</h2>
         <recent-uploads></recent-uploads>
     </section>
 </div>
+<div ng-if="ctrl.canUpload === false" class="unauthorised-message">You are not authorised to upload images</div>
```

Comment:
minor: should we provide information on how to resolve this? For example "You are not authorised to upload images. Send an email to admin@example.com if you think this is incorrect."

Reply:
Fixed